repository-management.md

```diff
@@ -0,0 +1,101 @@
+# Repository Management
+
+## Package Names
+
+Package names for the official `wp-cli` GitHub repository need to be discussed upfront with the [maintainers](https://github.com/orgs/wp-cli/teams/maintainers).
```

Comment:
`s/GitHub repository/GitHub organization`

Can you explain why package naming is important? What's an overview to our decision-making criteria before naming?

Maybe something like this:

> Before creating a new repository, it's important to discuss the repository first. This discussion helps establish consensus on what purpose the codebase serves, and where it lives best in the WP-CLI ecosystem.

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/1deaf59f20647653ae2155165db88ebd46bb9cc6 & https://github.com/wp-cli/handbook/pull/127/commits/7aebd670232e4516648e7a0ebdbdc265920f0e0f